installer/install-ntt.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+version="0.4"
+os="$(uname -s | tr "[:upper:]" "[:lower:]")"
+
+case "${os}" in
+darwin|linux)
+  os="$(echo "$os" | sed -e "s/\b\(.\)/\u\1/g")"
```

Comment:
remove this line.

Reply:
I'll change the download scheme, so we can get rid of this unfortunate line.